src/usr/local/www/diag_ping.php

```diff
@@ -76,6 +83,10 @@
 		if (preg_match('/[^0-9]/', $count)) {
 			$count = DEFAULT_COUNT;
 		}
+		$wait = $_REQUEST['wait'];
+		if (preg_match('/[^0-9]/', $wait)) {
```

Comment:
If your input validation above is better, this `preg_match` is unnecessary. Even if it does match the existing test above it, it's better to improve the code than to match the existing format. 

This whole statement could be one line:
```
$wait = (empty($_REQUEST['wait'])) ? DEFAULT_WAIT : $_REQUEST['wait'];
```